README_OSE.md

```diff
@@ -128,5 +144,17 @@ the directory as the inventory) use the -i option for ansible-playbook.
 On the master host:
 ```sh
 systemctl restart openshift-sdn-master
-openshift ex router --create=true --credentials=/var/lib/openshift/openshift.local.certificates/openshift-client/.kubeconfig
+openshift ex router --create=true \
+  --credentials=/var/lib/openshift/openshift.local.certificates/openshift-client/.kubeconfig \
+  --images='docker-buildvm-rhose.usersys.redhat.com:5000/openshift3_beta/ose-${component}:${version}'
+```
+
+#### Create the default docker-registry
+On the master host:
+```sh
+mkdir /var/lib/openshift/docker-registry
```

Comment:
Actually I'm not sure how to set up the volume since it needs to be available on whichever node the registry lands on. I thought it'd be nice to have the registry volume persistent but this probably doesn't achieve it.